LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -49,7 +49,11 @@
  Any help would be greatly appreciated.
  */
 
+#if defined(LTH_APP_EXTENSIONS)
```

Comment:
That's the expected way. To do it with a `#define` it should be coded with `#ifdef LTH_APP_EXTENSIONS`, but I think the current approach is cleaner as you may group all macros in the extension settings, not affecting the code.